data/json/items/tool/cooking.json

```diff
@@ -663,6 +663,15 @@
     ],
     "magazine_well": 2
   },
+  {
+    "id": "multi_cooker_filled",
+    "type": "TOOL",
+    "copy-from": "multi_cooker",
+    "name": { "str": "multi cooker - cooking" },
+    "description": "A professional-grade multi-cooker, with a battery slot for camping trips or tailgating. There is a dish in it.",
+    "//": "Don't know how to force it to lose qualities. Specifying empty array here seems not to work.",
```

Comment:
You can not use `copy-from` and just put the full item definition.

Reply:
> ```json
> "delete": { "qualities": [ [ "COOK", 3 ] ] },
> ```

Did not worked for qualities.
I made full item definition instead.

Reply:
See https://github.com/CleverRaven/Cataclysm-DDA/pull/37804

Reply:
@ZhilkinSerg 's PR is landed, so you can redo this on top of his new feature.

Reply:
Done:

![image](https://user-images.githubusercontent.com/16213433/75088923-7bc7be80-5564-11ea-9ba6-ad62ecda5728.png)